lochness/utils/source_check.py

```diff
@@ -114,6 +123,8 @@ def check_list_all_redcap_subjects(project_name: str,
 
     df = pd.DataFrame(content_dict_list)
     if len(df) > 1:
+        df.columns = ['subject', '_', 'consent_date']
```

Comment:
I don't see the use of these three columns in `get_xnat_credentials_from_config()` at all. Can you comment?

Reply:
The `source_check.py` screens each data sources like XNAT, REDCap, and Box etc. These columns you highlighted extracts subject and corresponding consent date from the REDCap, which is not directly related with the `get_xnat_credentials_from_config`.